client/my-sites/media-library/data-source.jsx

```diff
@@ -62,6 +63,11 @@ export class MediaLibraryDataSource extends Component {
 				label: translate( 'Pexels free photos' ),
 				icon: <PexelsIcon className="gridicon" />, // eslint-disable-line wpcalypso/jsx-classname-namespace
 			} );
+			sources.push( {
+				value: 'openverse',
+				label: translate( 'Openverse free photos' ),
```

Comment:
This could probably be better. It's currently just a duplicate of the above.

Reply:
Maybe `Openverse images`? Or do you think it's important to highlight they are free?